src/app/cli/src/commands/pull_command.rs

```diff
@@ -412,7 +304,7 @@ enum ProgressStyle {
     Bar,
 }
 
-struct PrettyIngestProgress {
+pub(crate) struct PrettyIngestProgress {
```

Comment:
Minor: move to a separate file, as it's getting reused

Reply:
I separated push and poll on my current branch and removed this export